packages/adapter/addon/rest.ts

```diff
@@ -309,14 +320,15 @@ class RESTAdapter extends Adapter.extend(BuildURLMixin) {
   }
 
   set fastboot(value) {
-    return (this._fastboot = value);
+    this._fastboot = value;
```

Comment:
We could take this private property off the class if you so desired with a WeakMap